.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,66 @@
+---
+# yamllint disable rule:line-length
+# Automatically runs on PRs to ensure the changes meet requirements and pass tests.
+name: PR Validation
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [master]
+    types: [opened, reopened, synchronize]
+  pull_request_target:
+    branches: [master]
+    types: [opened, reopened, synchronize]
```

Comment:
I don't think we need these anymore. In Commerce Dependabot triggers the `pull_request` event. I have a ticket for removing it in Referral

Reply:
Let me know if it looks right now